extensions/src/Smithy.Identity.Abstractions/Auth/IAuthSchemeOption.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Smithy.Identity.Abstractions
+{
+    /// <summary>
+    /// The output from the authentication scheme resolver. The resolver returns a list of these, in the
```

Comment:
nit: it's a little bit unclear what "these" is referring to when you say "returns a list of these". Can you specify what you're referring to?

Reply:
Do you have a suggestion? These are coming from the SRA book.

Reply:
Maybe instead of "these" just say The resolver returns a list of AuthSchemeOptions?

Reply:
Updated in https://github.com/aws/aws-sdk-net/pull/3382/commits/01b7a7cd84e858b5bfb3d6300a2bfa3b35e4aea1